packages/@o3r/schematics/src/modules/modules.helpers.ts

```diff
@@ -77,22 +75,19 @@ export const isOtterTag = <T extends string | undefined>(keyword: any, expect?:
 export interface AvailableModuleOptions extends PackageManagerOptions {
   /**
    * Determine if only the packages that are NOT installed should be returned
-   *
    * @default false
    */
   onlyNotInstalled?: boolean;
 
   /**
    * Npm Registry to fetch to get the package information
-   *
    * @default registry.npmjs.org
    */
   npmRegistryToFetch?: string;
 }
 
 /**
  * Get Available Otter modules on NPMjs.org
- *
```

Comment:
don't forget to have a look at the CodeQL

Reply:
Not sure to get the comment :).
What CodeQL you are talking about? Is is a non-accepted warning part of this PR change?

Reply:
![image](https://github.com/AmadeusITGroup/otter/assets/110408057/c3843967-5ca9-41b5-ba3c-4e26380674df)

Reply:
you should be able to fix it by replacing your promisify(exec) with promisify(execFile) in your np,Search method

Reply:
Not sure to understand your proposal, except if you want to create a file containing the command to run (which is just a dirty way to by-pass the check but does not resolve the original issue).
Here for me the warning from CodeQL makes sense (as we do use schematic input in a command line execution) and approving on our side make sense as well if we consider the impact accepted. Of course if it is not the case, we can find another way to do it.

Reply:
I will do the change but this following point is a bit scary: https://nodejs.org/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows
Are you confident that the `execFile` for `.bin` executable are correctly working on Windows?